src/RTCRtpSender.ts

```diff
@@ -0,0 +1,50 @@
+import {NativeModules} from 'react-native';
+import MediaStreamTrack from './MediaStreamTrack';
+import RTCRtpCapabilities, { getCapabilities } from './RTCRtpCapabilities';
+
+const {WebRTCModule} = NativeModules;
+
+
+export default class RTCRtpSender {
+    _id: string;
+    _track: MediaStreamTrack | null = null;
+    _peerConnectionId: number;
+
+    constructor(info: { peerConnectionId: number, id: string, track?: MediaStreamTrack }) {
+        this._peerConnectionId = info.peerConnectionId;
+        this._id = info.id;
+        if (info.track)
+            this._track = info.track;
+    }
+
+    replaceTrack(track: MediaStreamTrack | null): Promise<void> {
+        return new Promise<void>((resolve, reject) => {
+            WebRTCModule.senderReplaceTrack(this._peerConnectionId, this._id, track ? track.id : null, (successful, data) => {
```

Comment:
Make this return a native promise instead. Callbacks in this library are an ancient thing we try to get rid off.